backend/spec/requests/memos_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+RSpec.describe 'Memos' do
```

Comment:
IMO

テスト対象となるコントローラーを指定する方が一般的かなと思いました〜
```suggestion
RSpec.describe MemosController do
```

Reply:
確かにテスト対象となるものを記載しますもんね。
今回で言うとコントローラーのテストなのでご指摘の通りMemosControllerが適切だと僕も思いました！